cmd/main.go

```diff
@@ -96,10 +96,14 @@ func createUsers() *cobra.Command {
 			if err != nil {
 				return err
 			}
+			appID, err := core.HexToByteArray(cfg.AppID)
+			if err != nil {
+				return err
+			}
 
 			var users []User
 			for i := 0; i < userCount; i++ {
-				newWallet, err := wallet.NewWallet(core.StringToByteArray(cfg.AppID))
```

Comment:
Eew ... So StringToByteArray treats the input string as ascii or utf8 encoded, while HexToByteArray treats it as hex, which is what we want.

Reply:
yeah, no wonder I wasn't using this, it was a tech debt from a previous version, thankfully it's no more.